udata_hydra/utils/file.py

```diff
@@ -31,23 +31,29 @@ def read_csv_gz(file_path):
     return temp_file
 
 
-async def download_resource(url: str, headers: dict, ignore_size: bool = False) -> BinaryIO:
+async def download_resource(
+    url: str,
+    headers: dict,
+    file_format: str,
+    ignore_size: bool = False
```

Comment:
`file_format` with ignore_size logic could be computed to use a single `max_size_allowed` param, that could be None?
It would make download resource slightly more focused?